tracing/zipkin/src/test/java/io/helidon/tracing/zipkin/ZipkinBaggadeTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.tracing.zipkin;
+
+import io.helidon.config.Config;
+import io.helidon.tracing.HeaderConsumer;
+import io.helidon.tracing.HeaderProvider;
+import io.helidon.tracing.Span;
+import io.helidon.tracing.Tracer;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeMap;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+class ZipkinBaggadeTest {
+
+    public static final String KEY = "fubar";
+    public static final String VALUE = "1";
+
+    @BeforeAll
+    static void startTheServer() {
```

Comment:
wrong method name
also you do not need to do this in a before all, when you have a single test method.
Even if you do, do not register the tracer as global, as that may conflict with other tests. It is better to create a tracer and store it in a field (or just local variable within a test).

Reply:
Addressed